api/integrations/lead_tracking/hubspot/tasks.py

```diff
@@ -27,41 +26,37 @@ def track_hubspot_lead(user_id: int, organisation_id: int = None) -> None:  # ty
             id=organisation_id
         )
 
-    hubspot_lead_tracker.create_lead(**create_lead_kwargs)
+    hubspot_lead_tracker.create_organisation_lead(**create_lead_kwargs)
 
 
 @register_task_handler()
-def update_hubspot_active_subscription(subscription_id: int) -> None:
+def track_hubspot_user_contact(user_id: int) -> None:
```

Comment:
Again, this name isn't very clear to me. Perhaps it stems from the naming in the lead tracker? 

For example, I see that we're calling `hubspot_lead_tracker.create_user_hubspot_contact` - this would be better named (I think) as `hubspot_lead_tracker.create_hubspot_contact_for_user` ?